utilities/zookeeper/tests/testthat/test-reichlab-formatter.R

```diff
@@ -1,26 +1,41 @@
 test_that("format_predictions_for_reichlab_submission works", {
     pcard <- tibble(
-      ahead = rep(c(1, 2, 3), each = 5),
+      ahead = rep(c(1, 2, 3, 4), each = 5),
       geo_value = "pa",
-      quantile = rep(c(0.1, 0.4, 0.5, 0.6, 0.9), 3),
-      value = seq(1, 15),
+      quantile = rep(c(0.1, 0.4, 0.5, 0.6, 0.9), 4),
+      value = 1:20,
       forecaster = "a",
       forecast_date = as.Date("2020-01-02"),
       data_source = "source",
-      signal = rep(c("confirmed_incidence_num", "confirmed_incidence_num", "deaths_incidence_num"),
+      signal = rep(c("confirmed_incidence_num", "confirmed_incidence_num",
+                     "deaths_incidence_num", "confirmed_admissions_covid_1d"),
                    each = 5),
-      target_end_date = rep(as.Date(c("2020-01-09", "2020-01-16", "2020-01-23")), each = 5),
-      incidence_period = "epiweek"
+      target_end_date = rep(as.Date(c("2020-01-09", "2020-01-16", "2020-01-23", "2020-01-23")), each = 5),
```

Comment:
Might need to be 2020-01-30 for the 4-wk-ahead?

Reply:
or 2020-01-06 for 4 day ahead?

Reply:
It appears that it should be the latter.  This highlights that the formatter is not validating the `target_end_date`s.

Reply:
Additionally, the formatter is not validating that the `geo_value`s meet its expectations; e.g., it could get state fips codes and attempt to convert them "from abbrevations" to fips codes, resulting in NAs.  The existing partial guard against this is the check for the `"predictions_cards"` class, but, e.g., `as.predictions_cards` looks like it may still allow a user to encounter this case.

Reply:
Addressed.  Validation issue noted in #127.